src/wallet/wallet.cpp

```diff
@@ -102,14 +102,26 @@ std::string COutput::ToString() const
     return strprintf("COutput(%s, %d, %d) [%s]", tx->GetHash().ToString(), i, nDepth, FormatMoney(tx->tx->vout[i].nValue));
 }
 
+/** A class to identify which pubkeys a script and a keystore have in common. */
 class CAffectedKeysVisitor : public boost::static_visitor<void> {
 private:
     const CKeyStore &keystore;
     std::vector<CKeyID> &vKeys;
 
 public:
+    /**
+     * @param keystoreIn The CKeyStore that is queried for the presence of a pubkey.
+     * @param vKeysIn A vector to which a script's pubkey identifiers are appended if they are in the keystore.
```

Comment:
nit: could declare these [in]

Reply:
https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam

Reply:
Thanks, fixed